.github/workflows/review-pr.yml

```
@@ -0,0 +1,65 @@
+name: Review PR
+on:
+  pull_request:
+    paths:
+      - 'recipes/**'
```

rationale: [ignore draft PRs](https://github.com/orgs/community/discussions/25722#discussioncomment-3248917)
```suggestion
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]
    paths:
      - 'recipes/**'
```